azure-servicefabric/azure/servicefabric/service_fabric_client_ap_is.py

```diff
@@ -10,20 +10,22 @@
 # --------------------------------------------------------------------------
 
 from msrest.service_client import ServiceClient
-from msrest import Configuration, Serializer, Deserializer
+from msrest import Serializer, Deserializer
+from msrestazure import AzureConfiguration
 from .version import VERSION
 from msrest.pipeline import ClientRawResponse
+import uuid
 from . import models
 
 
-class ServiceFabricClientAPIsConfiguration(Configuration):
+class ServiceFabricClientAPIsConfiguration(AzureConfiguration):
```

Comment:
Why did you use the Azure generator? Should not be needed and will change your entire set of dependencies.